dev/com.ibm.ws.microprofile.metrics.3.0_fat_tck/publish/tckRunner/tck/src/test/java/com/ibm/ws/microprofile/metrics/test/ArquillianLoadableExtension.java

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
```

Comment:
2020?

Reply:
This file is "copied". I believe we don't change the copyright for such files since it isn't _new_.